### PR TITLE
move nuget packages under the repo for build scripts as well

### PR DESCRIPTION
**Pull request by brthor**
@eerhardt

### CODE REVIEW

**Comment by eerhardt** [comment]
test CentOS7.1 x64 Debug Build

File: build_projects/dotnet-host-build/build.ps1 [hunk 1]
```diff
@@ -26,6 +26,7 @@ if($Help)
 }
 
 $env:CONFIGURATION = $Configuration;
+$env:NUGET_PACKAGES = "$RepoRoot\.nuget\packages"
```
Comment (eerhardt) [inline build_projects/dotnet-host-build/build.ps1]: I think this needs to go after setting the $RepoRoot.

**Comment by brthor** [comment]
fixed.

**Comment by eerhardt** [comment]
:shipit:

**Pull request merged**